.github/workflows/basic_cli_build_release.yml

```diff
@@ -1,31 +1,45 @@
 on:
   workflow_dispatch:
-  release:
-    types: [edited]
```

Comment:
Turns out this did not trigger when new files are uploaded to the nightly releases.